proposals/CHAOSS-Improving-Open-Source-Sustainability.md

```diff
@@ -0,0 +1,113 @@
+
+2. **How is this proposal innovative -- what distinguishes it from other related work?** The proposal builds from the solid base that the CHAOSS project provides. It is distinguished from other related work as the CHAOSS project is the leading open source project focused precisely on open source community health and sustainability. Our position with the open source landscape sets us apart as a community with members from industry, academia, and funding agencies all focused on improving the health and sustainability of open source communities. 
+
+The particular projects are innovative as they advance prior efforts within the CHAOSS project and start to place them in front of people who want easy and quick views of open source community health and sustainability, as informed by years of community research. The projects will move the CHAOSS work from articulated documents to activities in practice available for all. We’ve built the base, now we are putting the base into practice for broad audiences. 
+
+As a whole, each project in this proposal contributes to the scaling of CHAOSS standards and software so they can answer larger questions, contribute to the development of ecosystem level maps, and be more easily applied by a larger community of open source stakeholders. 
+
+3. **Who is your doer -- who will execute the proposed work?** The doers include the authors of this proposal as well as CHAOSS community members engaged in different projects described here. Further, funds would be used to support two graduate students and one graphic design consultant. 
+
+4. **How might this work be sustained long-term after an initial seed grant?** Members of the proposing team have been long-term contributors to open source community health and sustainability efforts. Our work has been funded by the Alfred P. Sloan Foundation, the Ford Foundation, the Chan Zuckerberg Initiative, Mozilla, the National Science Foundation, and several for-profit organizations. We are experienced in partnering with external funding organizations - understanding the goals of funding for all the parties involved. Our efforts not only include the aforementioned projects in this proposal but the ongoing securing of external financial support to ensure the longevity of the CHAOSS project. 
+
+### Resources needed
+
+Amount requested (please specify payment breakdown by year and quarter, and by recipient if
+there are multiple): $400,000 ($320,000 on start, $120,000 in Q3 FY22)
+
+Project Team Leads: $90,000
+Graphic Designer: $80,000
+Software Community Manager: $80,000
+Graduate Student: $65,000
+Hardware for Metrics Models Deployment: $15,000 
+Hardware for Ecosystem Machine Learning Analysis $40,000
+Travel: $30,000
+Overhead costs (10%): $40,000
+
+Total:  $440,000
+
```

Comment:
Timeline and outputs (we will do XXX, YYY, etc.) and outcomes (long-term effects of work)

Reply:
I put the outputs and outcomes at the end of the project summary, and the timeline down here before I read your comment, @epicfaace . I'm happy to move everything to here, but thought I would let you read it first.